justfile

```diff
@@ -15,7 +15,7 @@ watch:
     @echo "Watching slides"
     typst w slides.typ
 
-# Compile and Run the C code
-c:
+# Compile the C code in "code/"
+check:
     @echo "Compiling and Running the C code"
-    @echo "TODO!"
\ No newline at end of file
+    ./compile-all.sh
```

Comment:
Change to
```
check:
    #!/usr/bin/env bash
    echo "compiling all *.c code to output"
    rm -r ./output;mkdir ./output
    for FILE in $(ls ./solutions/)
    do
	    cc ./solutions/$FILE -o ./output/$FILE.out
    done
```

Changes:

- Using `/usr/bin/env bash` is more failsafe than `/bin/bash`
- `cc` is more failsafe than `clang`

Reply:
Okay